data/mods/Dark-Skies-Above/DESIGN.md

```diff
@@ -2,42 +2,41 @@
 
 This document is intended to not only guide new potential contributors, but also help get my own thoughts on how the game will progress in text. Assume spoilers beyond this point!
 
+# Contributing
 
-# Summary of Intent and Content
-
-Mechanically, Dark Skies Above began as a project to demonstrate how you can push the game engine to present different stories. Unlike the base game, a narrative arc is assumed: the player will want find a way to make occupation of Earth too costly for the aliens to continue, or die trying. This is, of course, by no means an "easy" or "short" process, nor is it necessarily a requirement. This is done largely to compensate for the fact that Dark Skies does not play well with other mods or some aspects of active development, thus needing a tighter experience.
+Since Dark Skies is distributed with the base game, contributions are welcome. If you have any questions, you can contact me on Discord viazz `@ephemeral_storyteller` through the official developer or official community severs.
```

Comment:
Ephemeral Storyteller is no longer the maintainer, and should probably be removed from "contacting"